content/en/docs/security/best-practices.md

```diff
@@ -112,7 +112,7 @@ The recommendations below are based on Flux latest version.
 - Ensure Secret Decryption is enabled and secrets are not being held in Flux Sources in plain-text.
   <details>
     <summary>Rationale</summary>
-    Helm and Kustomize Controllers have auto decryption mechanisms that can decrypt cipher texts on-demand at reconciliation time. That enables for credentials (e.g. passwords, tokens) and sensitive information to be kept encrypted in the sources.    
+    The Kustomize Controller have an auto decryption mechanism that can decrypt cipher texts on-demand at reconciliation time. That enables for credentials (e.g. passwords, tokens) and sensitive information to be kept encrypted in the sources.    
```

Comment:
Should we also make mention of it making use of SOPS? Something like

> [..] using an embedded implementation of [SOPS].

Reply:
Added the complement accordingly, PTAL.